src/main/kotlin/org/radarbase/redcap/util/RedCapClient.kt

```diff
@@ -166,6 +171,18 @@ open class RedCapClient(private val redCapInfo: RedCapInfo) {
             }
         }
 
+    fun mergeMaps(list: List<Map<String, String>>): Map<String, String> {
+        val mergedMap = mutableMapOf<String, String>()
+        for (map in list) {
+            for ((key, value) in map) {
+                if (!mergedMap.containsKey(key) && value != null) {
```

Comment:
Should we check for blank values too (as i remember i noticed sometimes the subject id from redcap was `""` in the json response)?